druid-shell/src/hotkey.rs

```diff
@@ -109,7 +109,7 @@ impl HotKey {
 
     /// Returns `true` if this [`KeyboardEvent`] matches this `HotKey`.
     ///
-    /// [`KeyboardEvent`]: keyboard_types::KeyEvent
+    /// [`KeyEvent`]: KeyEvent
```

Comment:
The text above as well, now the link doesn't match. And I'm very puzzled why this isn't being caught in CI, maybe we don't have our warnings tuned properly.

Reply:
Good point, let me look at this locally. I'm not very good with github edits.